web_responsive/views/web.xml

```diff
@@ -12,7 +12,7 @@
               name="App Drawer - Web Client"
               >
 
-        <xpath expr="//div[@contains('class', 'o_sub_menu')]" position="replace" />
```

Comment:
Sorry I think my suggestion was misunderstood - I should have caught it too, my bad. This is what we want:

```
//div[hasclass('o_sub_menu')]
```

[Examples](https://github.com/odoo/odoo/search?utf8=%E2%9C%93&q=%22hasclass%28%22&type=)